### DISCUSSION TITLE
Should the D:M:S printout of both location coordinates include decimal places for seconds?

### DISCUSSION THREAD

**Discussion by rileymoore13** [MRU-F22-CS1/instructions#7] [Q&A] [answered]
Question in title. Are integers required for the whole readout? Some online sources show seconds with a few decimal places.

**Comment by jpratt-mru** [comment, marked as answer]
Take a boo under point 5 under [Programming Requirements](https://github.com/MRU-F22-CS1/instructions/tree/main/assign2#programming-requirements) and the [Coordinate conversion](https://github.com/MRU-F22-CS1/instructions/tree/main/assign2#coordinate-conversion-from-decimal-notation-to-dms) section.

**Reply by rileymoore13** [reply]
Thank you!
